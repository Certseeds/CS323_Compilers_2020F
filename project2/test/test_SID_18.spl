struct AST {
    int x1;
    float x2;
    char x3;
    struct AST next;
};