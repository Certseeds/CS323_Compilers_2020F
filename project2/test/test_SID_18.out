Error type 16 at Line 5: struct AST haven't define
